From 0c980a59265fad9bf99666807db2bc0243b3012f Mon Sep 17 00:00:00 2001
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Thu, 15 Aug 2013 16:34:33 +0100
Subject: [PATCH] Fix broken 'default' case in switch statement

For https://bugzilla.redhat.com/show_bug.cgi?id=922639

The 'default' case somehow got placed on the wrong line,
leading to unreachable code.

Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
(cherry picked from commit 482108c970ecff1976826f686e42525a076352f5)
---
 libvirt-sandbox/libvirt-sandbox-config-interactive.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/libvirt-sandbox/libvirt-sandbox-config-interactive.c b/libvirt-sandbox/libvirt-sandbox-config-interactive.c
index 6d0e1e8..6fbb1c6 100644
--- a/libvirt-sandbox/libvirt-sandbox-config-interactive.c
+++ b/libvirt-sandbox/libvirt-sandbox-config-interactive.c
@@ -91,12 +91,11 @@ static void gvir_sandbox_config_interactive_set_property(GObject *object,
     GVirSandboxConfigInteractivePrivate *priv = config->priv;
 
     switch (prop_id) {
-
-    default:
     case PROP_TTY:
         priv->tty = g_value_get_boolean(value);
         break;
 
+    default:
         G_OBJECT_WARN_INVALID_PROPERTY_ID(object, prop_id, pspec);
     }
 }
